src/tqec/templates/atomic/rectangle.py

```diff
@@ -90,6 +91,23 @@ def to_dict(self) -> dict[str, ty.Any]:
     def expected_plaquettes_number(self) -> int:
         return 2
 
+    def get_midline_plaquettes(
+        self, horizontal: TemplateOrientation = TemplateOrientation.HORIZONTAL
```

Comment:
```suggestion
        self, orientation: TemplateOrientation = TemplateOrientation.HORIZONTAL
```
WARNING: should also be changed in the function body if changed here.